animated-base/src/main/java/com/facebook/imagepipeline/animated/impl/AnimatedImageCompositor.java

```diff
@@ -98,18 +98,25 @@ public void renderFrame(int frameNumber, Bitmap bitmap) {
     // Iterate from nextIndex to the frame number just preceding the one we're trying to render
     // and composite them in order according to the Disposal Method.
     for (int index = nextIndex; index < frameNumber; index++) {
-      frameInfo = mAnimatedDrawableBackend.getFrameInfo(index);
+      AnimatedDrawableFrameInfo frameInfo = mAnimatedDrawableBackend.getFrameInfo(index);
```

Comment:
Just a few small thing here. Instead of create a AnimatedDrawableFrameInfo reference when looping, can you move it outside of the loop?